app/src/main/res/drawable/buy_button_background.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <solid android:color="#FF82A9"/>
```

Comment:
@SergeiMikhailovskii move hex color to `values/colors.xml`. It would be easier to maintain all your application colors later.